addons/crm/wizard/crm_lead_lost.py

```diff
@@ -9,20 +9,25 @@ class CrmLeadLost(models.TransientModel):
     _name = 'crm.lead.lost'
     _description = 'Get Lost Reason'
 
+    lead_ids = fields.Many2many('crm.lead')
     lost_reason_id = fields.Many2one('crm.lost.reason', 'Lost Reason')
     lost_feedback = fields.Html(
         'Closing Note', sanitize=True
     )
 
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
+        defaults.setdefault('lead_ids', self._context.get('active_ids', []))
```

Comment:
Also I think the favored way of using the context is "self.env.context" and not "self._context".

Reply:
Yep, fixed these tests to just use the leads field. Note the very last test in that file behaving very weirdly